phpBB/styles/prosilver/theme/pm.css

```diff
@@ -0,0 +1,64 @@
+/* -------------------------------------------------------------- /*
+    Styles for the refactored PM module
+/* -------------------------------------------------------------- */
+
+.pm-list {
+	font-size: 1rem;
+	font-weight: 400;
+	line-height: 1.5;
+	letter-spacing: 0.04em;
+	margin: 0;
+	padding: 0;
+	list-style-type: none;
+}
+
+.pm-item {
+	border-left: 5px solid transparent;
+	position: relative;
+	display: flex;
+	overflow: hidden;
+	justify-content: flex-start;
+	align-items: center;
+	padding: 16px;
+}
+
+.pm-unread {
+	border-left: 5px solid #d41142;
+}
+
+.pm-primary,
+.pm-secondary {
+	display: block;
+}
+
+.pm-primary {
+	line-height: 1.5;
+}
+
+.pm-secondary {
+	font-size: 0.875rem;
+	font-weight: 400;
+	line-height: 1.25;
+	text-decoration: inherit;
+	letter-spacing: 0.04em;
+	text-transform: inherit;
+}
+
+.pm-secondary-item {
+	line-height: 1.2857142857;
+	padding-right: 1rem;
+}
+
+/*	colours and backgrounds  */
+
+.pm-item {
+	background-color: #dbe9f0;
+}
+
+.pm-item:nth-child(even) {
+	background-color: #edf4f7;
+}
+
+pm-unread-pm-title {
```

Comment:
Check above comment for how to handle this, but it will need to be in colours.css